### ISSUE TITLE
Drupal 8.7 is released

### ISSUE THREAD

**Issue by Natkeeran** [Islandora-CLAW/CLAW#1110]
[Drupal 8.7](https://www.drupal.org/blog/drupal-8-7-0) has been released.

We need to test and update Drupal to 8.7 the [drupal project composer.json](https://github.com/Islandora-CLAW/drupal-project/blob/8.x-1.x/composer.json#L24).

This issue that was throwing exceptions in ansible and that necessitated https://github.com/Islandora-Devops/claw-playbook/blob/master/post-install.yml#L13 has been fixed.

**Comment by mjordan** [comment]
Can we test #1035 as part of this?

**Comment by Natkeeran** [comment]
@dannylamb It seems like there is more to that than simply changing the drupal version.  Also, wondering if you want to pull in changes from https://github.com/drupal-composer/drupal-project.

**Comment by dannylamb** [comment]
@Natkeeran Yes, please. I usually pull in the latest from upstream whenever I'm stomping around in there.  What are you running into?

**Comment by Natkeeran** [comment]
@dannylamb 
Here is an attempt: https://github.com/Islandora-CLAW/drupal-project/tree/drupal_87

1) how to test this without being able to update composer package
2) how does the composer.lock file gets updated?

**Comment by Natkeeran** [comment]
@mjordan 

This issue is not fixed, thus 1035 is not fixed: https://www.drupal.org/project/drupal/issues/2350939

The ansible issue is fixed.

**Comment by dannylamb** [comment]
Just `vagrant up`d with the 8.7 release of `drupal-project`.  Red 5, I'm going in.

**Comment by dannylamb** [comment]
Alright, so, a bit of a red herring.  It looked like even with the configuration set to expose `/media/*` urls, the REST endpoints were `/media/*/edit`, which wrecked _everything_.  So I went through and refactored and put url generating function is IslandoraUtils and just call those from everywhere.  Still no dice.  _Then I cleared my cache and everything worked just fine_.

So the solution to this is to just add a cache clear after setting the variable in claw-playbook.

__BUT__ not all is lost, as that refactor actually fixes https://github.com/Islandora-CLAW/CLAW/issues/1095 :champagne: 

I'll throw up PRs for things as I clean up.  That refactor means I have to go touch up all the tests.
